src/com/dotmarketing/cache/ContentTypeCacheImpl.java

```diff
@@ -22,8 +26,10 @@
     private final String primaryGroup = "StructureCache";
     private final String containerStructureGroup = "ContainerStructureCache";
     private final String structuresByTypeGroup = "StructuresByTypeCache";
+    private final String recentGroup = "RecentCache";
```

Comment:
@wezell that recent cache is where we are planning to store the recently used content types.